react/components/EXPERIMENTAL_Table/DataTable/Cell.tsx

```diff
@@ -48,22 +55,49 @@ const Cell: FC<CellProps> & CellComposites = ({
       z1: !sticky,
     }
   )
-  const Tag = header ? 'th' : 'td'
+  const tag = header ? 'th' : 'td'
+  const style = {
+    position: sticky ? 'sticky' : 'static',
+    width,
+  } as CSSProperties
+  const Container = sortable ? HoverableCell : DefaultCell
 
   return (
-    <Tag
-      {...events}
-      onClick={onClick}
-      style={{
-        position: sticky ? 'sticky' : 'static',
-        width,
-      }}
-      className={className}>
+    <Container tag={tag} onClick={onClick} style={style} className={className}>
```

Comment:
since the name of the prop is the same as the value, you can spread.

```jsx
  const containerProps = { tag, onClick, style, className }
  <Container {...containerProps}>
```

Reply:
a lot cleaner. done!